eslint/babel-eslint-parser/src/convert/convertAST.js

```diff
@@ -95,9 +100,9 @@ function convertNodes(ast, code) {
 function convertProgramNode(ast) {
   ast.type = "Program";
   ast.sourceType = ast.program.sourceType;
-  ast.directives = ast.program.directives;
   ast.body = ast.program.body;
   delete ast.program;
+  delete ast.errors;
```

Comment:
Extra property that ESLint doesn't use (though we are planning to implement a top-level `recoverableErrors` property - you can find more info [here](https://github.com/eslint/rfcs/tree/master/designs/2019-recoverable-error-handling)).